trigger/sqs/README.md

```diff
@@ -0,0 +1,28 @@
+# AWS SQS Trigger
+
+The SQS trigger provides your Flogo application the ability to read data from SQ. 
+
+## Installation
+
+### Flogo CLI
+
+```bash
+flogo install github.com/project-flogo/aws-contrib/trigger/sqs
+```
+
+## Configuration
+
```

Comment:
Add the note on how to configure the SDK, ```To configure AWS credentials see [configuring-sdk](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html)```

We'll want to add that to the S3 readme as well.